src/mixins/textfield-mixin.js

```diff
@@ -0,0 +1,58 @@
+export default {
+  props: {
```

Comment:
Could you add the following HTML input attributes?
* `min`, `max` and `step`: all of these of type number
* `readonly`: boolean attribute.